resin_release_tool/releaser.py

```diff
@@ -21,6 +22,62 @@ def get_canaries(self):
             if tag['tag_key'] == 'CANARY']
         return canaries
 
+    def get_tags_per_device(self):
+        all_devices = self.get_all_devices()
+        tags = self.models.tag.device.get_all_by_application(self.app_id)
+        device_dict = {}
+        for device in all_devices.values():
+            device_id = device['id']
+            device_dict[device_id] = {
+                'uuid': device['uuid'],
+                'tags': {},
+            }
+        for elem in tags:
+            device_id = elem['device']['__id']
+            tag_key = elem['tag_key']
+            value = elem['value']
+            device_dict[device_id]['tags'][tag_key] = value
+        tags_per_device = [device_dict[device] for device in list(device_dict.keys())]
+        return tags_per_device
+                    
+    def get_app_env_vars(self):        
+        all_vars = self.models.environment_variables.application.get_all(int(self.app_id))
+        return [{'id':var['id'], var['name']:var['value']} for var in all_vars]
```

Comment:
Oh the original line got refactored but https://github.com/mobilityhouse/resin-release-tool/pull/10#discussion_r256840952 still applies